install/README.md

```diff
@@ -245,6 +247,56 @@ For the scale factor and bias either a value can be given directly or an express
 
 For a second order polynomial response, the output is expected to be `Y = a * X + b` where `X` is normally the input voltage, and Y the corrected signal. The terms `a` and `b` are the factor and bias respectively. The gain adjustments (`a'`, `b'`) update this via `Y = (a * a') * X + (b + b')`
 
+#### _COMPONENTS_ ####
+
+Sensor model component descriptions. The type is generally of the form "Accelerometer, Short Period Seismometer" etc.
+The number represents the order of sensor components, this generally maps to the sensor cable and how it is connected
+into the datalogger.
+Subsource is the general term used for labelling the sensor component and is usually the last character in the SEED channel convention.
+Dip and Azimuth are used to indicate the relative position of the sensor component within the sensor package and will be used with the
+overall sensor installation values to provide component dips and azimuths.
+
+| Field       | Description |
+| ----------- | ----------- |
+| _Make_      | Sensor make
+| _Model_     | Sensor model name
+| _Type_      | Sensor type
+| _Number_    | Sensor component offset
+| _Subsource_ | Sensor component label
+| _Dip_       | Internal dip of the compnent relative to whole sensor
+| _Azimuth_   | Internal azimuth of the compnent relative to whole sensor
+| _Types_     | A shorthand reference to the SEED type labels
+| _Response_  | A reference to the nominal StationXML response
+
+Example:
+
+    Make,Model,Type,Number,Subsource,Dip,Azimuth,Types,Response
+    Guralp,Fortis,Accelerometer,0,Z,-90,0,G,sensor_guralp_fortis_response
+    Guralp,Fortis,Accelerometer,1,N,0,0,G,sensor_guralp_fortis_response
+    Guralp,Fortis,Accelerometer,2,E,0,90,G,sensor_guralp_fortis_response
```

Comment:
this section to describe `components` is already present in the paragraph below (after "CALIBRATIONS"), shall one of the two be removed from the README?